public/server/db/cfbr.json

```diff
@@ -0,0 +1,3 @@
+{
+    "definition": "Commenting For Better Reach"
```

Comment:
Unfortunately, this already exists in the records. To make sure that an abbreviation doesn't already exist, you can look it up by searching on the site.

Reply:
I didn't know how this came in this PR as i have created other branch for these abbreves